CHANGELOG.md

```diff
@@ -1,3 +1,190 @@
+## Release 0.14.0
+
+Highlights:
+
+This release comes with two main new features:
+  * Plugins. Now you can create your http-based plugin to provide new metrics
+    and display them in Scope. You can read more about it and see some examples [here]().
+  * _Metrics on canvas_: Metrics are now displayed on the nodes and not just on
```

Comment:
Consistency: this prefix italicized and ends with a colon, but the previous prefix is styled normally and ends with a period.